packages/core/src/conversation/ConversationService/ConversationService.ts

```diff
@@ -384,11 +384,6 @@ export class ConversationService {
   }
 
   public async wipeMLSConversation(groupId: string): Promise<void> {
-    const isMLSConversationEstablished = await this.isMLSConversationEstablished(groupId);
-    if (!isMLSConversationEstablished) {
-      //if the mls group does not exist, we don't need to wipe it
-      return;
-    }
```

Comment:
This part was moved to `MLSService.wipeConversation` to make sure we always check if conversation exists before wiping it.